workspaces/mend/plugins/mend/src/hooks/index.tsx

```diff
@@ -5,5 +5,5 @@ export const useResize = (fn: () => void): void => {
     fn();
     window.addEventListener('resize', fn);
     return () => window.removeEventListener('resize', fn);
-  }, []);
+  }, [fn]);
```

Comment:
I think we need to check that fn is memoized 😅

Reply:
It does not break UI but I can add useCallback 😉
<img width="1742" alt="Screenshot 2025-02-12 at 16 29 49" src="https://github.com/user-attachments/assets/c8d86503-5265-476b-b8df-8a7bb8206a29" />

Reply:
it doesn't break anything but it will cause `addEventListener()` to be invoked every time the component re-renders

Reply:
agree

Reply:
@dariuszsobkowicz Would you like to work on this, I have no much context in this plugin and was just generation PR for tracking lint issues.

Reply:
Yes, I will push out the fixes

Reply:
@Sarabadu done